resolwe_bio/processes/reads_processing/cutadapt.yml

```diff
@@ -205,130 +216,131 @@
+    - name: adapters
+      label: Adapters
+      group:
+        - name: up_primers_file_fwd
+          label: 5 prime forward reads adapter file
+          type: data:seq:nucleotide
+          required: false
+        - name: down_primers_file_fwd
+          label: 3 prime forward reads adapter file
+          type: data:seq:nucleotide
+          required: false
+        - name: up_primers_file_rev
+          label: 5 prime reverse reads adapter file
+          type: data:seq:nucleotide
+          required: false
+        - name: down_primers_file_rev
+          label: 3 prime reverse reads adapter file
+          type: data:seq:nucleotide
+          required: false
+        - name: up_primers_seq_fwd
+          label: 5 prime forward reads adapter sequence
+          type: list:basic:string
+          required: false
+        - name: down_primers_seq_fwd
+          label: 3 prime forward reads adapter sequence
+          type: list:basic:string
+          required: false
+        - name: up_primers_seq_rev
+          label: 5 prime reverse reads adapter sequence
+          type: list:basic:string
+          required: false
+        - name: down_primers_seq_rev
+          label: 3 prime reverse reads adapter sequence
+          type: list:basic:string
+          required: false
+        - name: times
+          label: Times
+          type: basic:integer
+          required: false
+          default: 1
+          description: >
+            Remove up to COUNT adapters from each read.
+        - name: error_rate
+          label: Error rate
+          type: basic:decimal
+          default: 0.1
+          description: >
+            Maximum allowed error rate (no. of errors divided by the length of the matching region).
+        - name: min_overlap
+          label: Minimal overlap
+          type: basic:integer
+          default: 3
+          description: >
+            Minimum overlap for an adapter match
+        - name: match_read_wildcards
+          label: Match read wildcards
+          type: basic:boolean
+          required: false
+          default: false
+          description: >
+            Interpret IUPAC wildcards in reads.
+    - name: modify_reads
+      label: Modifying reads
+      group:
+        - name: leading
+          label: Leading quality
+          type: basic:integer
+          required: false
+          description: >
+            Remove low quality bases from the beginning. Specifies the minimum quality required to keep a base.
+        - name: trailing
+          label: Trailing quality
+          type: basic:integer
+          required: false
+          description: >
+            Remove low quality bases from the end. Specifies the minimum quality required to keep a base.
+        - name: crop
+          label: Crop
+          type: basic:integer
+          required: false
+          description: >
+            Cut the specified number of bases from the end of the reads
+        - name: headcrop
```

Comment:
Consider joining this with `crop` parmeter as it is done in CL interface. Explain the impact of `-` sign.

Reply:
Same as above